packages/bots/liquidator/src/justSubmit.ts

```diff
@@ -15,12 +23,12 @@ const account = privateKeyToAccount(config.adminPrivateKey as Hex);
 const publicClient = createPublicClient({
   batch: { multicall: { wait: 16 } },
   chain: mode,
-  transport: http(config.rpcUrl),
+  transport: fallback(config.rpcUrls.map((url) => http(url))),
```

Comment:
Won't this be a string that needs to be split first?

Reply:
I was doing this in config.ts
(https://github.com/ionicprotocol/monorepo/blob/18196d2ed7dc59e69e0ebbf8e9fbf19eeffd34d3/packages/bots/liquidator/src/config.ts#L8)

Reply:
https://github.com/ionicprotocol/monorepo/blob/18196d2ed7dc59e69e0ebbf8e9fbf19eeffd34d3/packages/bots/liquidator/src/config.ts#L9